test/test_model.py

```diff
@@ -3059,7 +3174,11 @@ class TestSecretClass(unittest.TestCase):
     def setUp(self):
         self.model = ops.Model(ops.CharmMeta(), _ModelBackend('myapp/0'))
 
-    def make_secret(self, id=None, label=None, content=None):
+    def make_secret(self,
+                    id: typing.Optional[str] = None,
+                    label: typing.Optional[str] = None,
+                    content: typing.Optional[typing.Dict[str,
+                                                         str]] = None):
```

Comment:
Just wondering: do we have our tooling set up wrongly that it's doing awkward things like this? Is the formatter line break set to less than the linter line break or something?

```suggestion
                    content: typing.Optional[typing.Dict[str, str]] = None):
```